### PR TITLE
Footer

### LINKED ISSUE #36: Donori projektu
Na webke, zrejme niekde dolu, by malo byť spomenuté, kto to platí:

- Fond pre transparentné Slovensko v Nadácii Pontis
- US Embassy 
- ďalšie kombinované zdroje zo Slovensko.Digital

### PR DESCRIPTION
**Pull request by eronisko**
Closes #36

### CODE REVIEW

**Comment by eronisko** [comment]
footer_laptop
footer_phone

**Comment by eronisko** [comment]
image
localhost-3000- iphone 5 1
localhost-3000- iphone 5 2

**Comment by eronisko** [comment]
image
localhost-3000- iphone 5 3

**Comment by eronisko** [comment]
image

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -26,10 +26,29 @@ em {
 .container {
   max-width: 1024px;
 }
-footer {
-  background-color: $gray-lighter;
+
+.hidden {
```
Comment (pavolzbell) [inline app/assets/stylesheets/application.scss]: radsej https://getbootstrap.com/docs/4.0/utilities/display/

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -29,44 +29,55 @@
...
+                  <strong>Ďakujeme za Váš záujem.</strong></br>
+                  Na zadanú adresu Vám čoskoro príde potvrdzovací email k odberu noviniek.
+                </p>
+              </div>
+            </div>
           </div>
 
           <div class="col-md-2">
-            <p class="pull-right icons">
+            <p class="md-pull-right icons">
```
Comment (pavolzbell) [inline app/views/layouts/application.html.erb]: `md-pull-right` toto funguje? radsej https://getbootstrap.com/docs/4.0/utilities/float/

**Pull request merged**